README.md

```diff
@@ -42,6 +42,27 @@ file to the `docs/` directory which is the root of the
 
 The build process requires Helm 3.
 
+### Testing the Helm Charts
+To test Helm Charts locally you will need to follow the next steps:
+
+1. Install docker, helm, kubectl, and [minikube](https://minikube.sigs.k8s.io/docs/start/), if you don't already have it on your local workstation.
+2. Start up minikube: `minikube start`
+3. Confirm minikube is up and running: `minikube status`
+4. List the existing pods in the cluster: `kubectl get pods`  (There should not be anything listed at this point.)
+5. Install the helm chart in any of these ways:
+    * From a copy of the source: `helm install iq {path/to/your/helm3-charts}/charts/nexus-iq` 
+    * From our production online repo: Add our helm repo locally as instructed at `https://sonatype.github.io/helm3-charts/`
+6. Install a server from the production chart: helm install iq sonatype/nexus-iq-server 
```

Comment:
this step 6 is only applicable if you're using the production repo. (part of the previous step 5.)

Reply:
you are right, I just updated the code 😄